azurelinuxagent/common/osutil/default.py

```diff
@@ -1270,6 +1270,7 @@ def _get_proc_stat():
             results = fileutil.read_file('/proc/stat')
         except (OSError, IOError) as ex:
             logger.warn("Couldn't read /proc/stat: {0}".format(ex.strerror))
+            raise ex
```

Comment:
Since you're raising here, this bubbles up to `get_total_cpu_ticks_since_boot`, which doesn't catch, which bubbles up to `__init__` of CpuCgroup (not caught), and to `_update_cpu_data` (not caught). Is this intended?